### PR TITLE
LAWS-262 ND updates

### PR DESCRIPTION
**Pull request by Aidancs**
refs: https://spurwork.atlassian.net/browse/LAWS-262

North Dakota uses the federal W4 form for its state form, the worker fills out a copy of the W4 and that is what is used for the state form. The federal W4 changed for 2020 so this updates the tax library. Sense we have never had anyone onboard in ND we don't need the 2019 stuff so I removed it.

### CODE REVIEW

File: src/Countries/US/NorthDakota/NorthDakotaIncome/V20200101/NorthDakotaIncome.php [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    {
+        if ($this->isUserClaimingExemption()) {
+            return 0.00;
+        }
+
+        $tax_amount = $this->getAdjustedWageAmount();
+        $tax_amount = $this->getTentativeAmount($tax_amount);
+        $tax_amount -= $this->tax_information->dependents_deduction_amount;
+        $tax_amount = $tax_amount > 0 ? $tax_amount / $this->payroll->pay_periods : 0;
+        $tax_amount += $this->tax_information->extra_withholding;
```
Comment (stevespur) [inline src/Countries/US/NorthDakota/NorthDakotaIncome/V20200101/NorthDakotaIncome.php:90]: so the extra withholding should be divided among the pay periods?

Comment (Aidancs) [reply src/Countries/US/NorthDakota/NorthDakotaIncome/V20200101/NorthDakotaIncome.php:90]: The tax amount is divided by the pay periods then the extra withholding is added to that.

Comment (stevespur) [reply src/Countries/US/NorthDakota/NorthDakotaIncome/V20200101/NorthDakotaIncome.php:90]: right, so is what they fill out on the form for extra withholding supposed to be for each pay period or is it supposed to be for the year?

Comment (Aidancs) [reply src/Countries/US/NorthDakota/NorthDakotaIncome/V20200101/NorthDakotaIncome.php:90]: each pay period
Screen Shot 2020-05-22 at 10 54 42 AM

Comment (stevespur) [reply src/Countries/US/NorthDakota/NorthDakotaIncome/V20200101/NorthDakotaIncome.php:90]: just double checking

Comment (Aidancs) [reply src/Countries/US/NorthDakota/NorthDakotaIncome/V20200101/NorthDakotaIncome.php:90]: I don't mind. I went to the form to ensure the correct answer. Double checking never hurts

**Pull request merged**